### PR TITLE
Who setup

### PR DESCRIPTION
**Pull request by alexsilaghi**
fixed some serialization issues

### CODE REVIEW

File: src/main/java/edu/stanford/protege/webprotege/app/ExecutionTimeAdvice.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+@Aspect
+public class ExecutionTimeAdvice {
+
+    private final static Logger log = LoggerFactory.getLogger(ExecutionTimeAdvice.class);
```
Comment (alexsilaghi) [inline src/main/java/edu/stanford/protege/webprotege/app/ExecutionTimeAdvice.java:13]: this still need fixing. Is not working properly yet.

File: src/main/java/edu/stanford/protege/webprotege/issues/EntityDiscussionThreadRepository.java [hunk 2]
```diff
@@ -51,27 +56,53 @@ public int getCommentsCount(@Nonnull ProjectId projectId,
 
     public int getOpenCommentsCount(@Nonnull ProjectId projectId,
                                     @Nonnull OWLEntity entity) {
+        List<Comment> comments = getOpenedCommentsForProject(projectId).get(entity.getIRI());
+        return comments == null ? 0 : comments.size();
+    }
+
+
+    public synchronized Map<IRI, List<Comment>> getOpenedCommentsForProject(ProjectId projectId) {
+        Map<IRI, List<Comment>> response = this.cache.get(projectId);
```
Comment (alexsilaghi) [inline src/main/java/edu/stanford/protege/webprotege/issues/EntityDiscussionThreadRepository.java:65]: create a comment cache per project

File: src/main/java/edu/stanford/protege/webprotege/issues/EntityDiscussionThreadRepository.java [hunk 3]
```diff
@@ -51,27 +56,53 @@ public int getCommentsCount(@Nonnull ProjectId projectId,
...
+
     public void saveThread(@Nonnull EntityDiscussionThread thread) {
         mongoTemplate.save(thread);
     }
 
     public void addCommentToThread(@Nonnull ThreadId threadId,
+                                   @Nonnull ProjectId projectId,
                                    @Nonnull Comment comment) {
         var query = createQueryForThread(threadId);
+        this.cache.remove(projectId);
```
Comment (alexsilaghi) [inline src/main/java/edu/stanford/protege/webprotege/issues/EntityDiscussionThreadRepository.java:97]: cache bust. when changing a comment on prj , then we bust the cache so when we fetch , we fetch them from the db.

File: pom.xml [hunk 4]
```diff
@@ -179,7 +179,7 @@
         <dependency>
             <groupId>edu.stanford.protege</groupId>
             <artifactId>webprotege-jackson</artifactId>
-            <version>0.9.0</version>
+            <version>0.9.3</version>
```
Comment (matthewhorridge) [inline pom.xml:182]: I don't see any changes in webprotege-jackson (missing PR?)

**Pull request closed without merging**